console/atest-ui/src/locales/en.json

```diff
@@ -1,4 +1,5 @@
 {
+    "lan": "en",
```

Comment:
I suggest using full spelling  `language` rather than abbreviations, as full spelling is clearer and more explicit. And please use the formal spell of `English`.

Reply:
The word is different.

![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/7c93eb10-0618-466c-8c0c-31c8b8dc089a)